strawberryfields/io/blackbird_io.py

```diff
@@ -83,11 +86,22 @@ def from_blackbird(bb):
 
 
 # pylint:disable=too-many-branches
-def from_blackbird_to_tdm(bb):
+def from_blackbird_to_tdm(bb: blackbird.BlackbirdProgram) -> TDMProgram:
+    """Convert a ``BlackbirdProgram`` to a ``TDMProgram``.
+
+    Args:
+        bb (blackbird.BlackbirdProgram): the input Blackbird program object
+
+    Returns:
+        Program: corresponding ``TDMProgram``
```

Comment:
The type hints of the function signature should agree with the Sphinx docs!
```suggestion
        TDMProgram: corresponding ``TDMProgram``
```